src/Native/SymSgdNative/CMakeLists.txt

```diff
@@ -18,4 +18,8 @@ endif()
 add_library(SymSgdNative SHARED ${SOURCES} ${RESOURCES})
 target_link_libraries(SymSgdNative PUBLIC ${MKL_LIBRARY})
 
+if(APPLE)
+    set_target_properties(SymSgdNative PROPERTIES INSTALL_RPATH "@loader_path;${MKL_LIB_PATH}")
```

Comment:
`MKL_LIB_PATH` points to our repo's `packages` folder. This isn't going to work for customers who install our NuGet package.

Reply:
We spoke offline about this. The current version has @loader_path that tells the loader to look into the same location as the referring binary, this will work for our users if did dotnet publish. We agreed to add another rpath that will also search in the nuget cache.

---
In reply to: [221632968](https://github.com/dotnet/machinelearning/pull/977#discussion_r221632968) [](ancestors = 221632968)